local-oldest-requirements.txt

```diff
@@ -1 +1 @@
-acme[dev]==0.26.0
+acme[dev]==0.29.0
```

Comment:
Ok FINAL ANSWER really this time: it DOES have to be `-e acme[dev]` and then changed back after the release. I know, I know.
```suggestion
-e acme[dev]
```

Reply:
:)